[2024-03-27T00:00:00.257+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_catchup_backfill_v02.first_task scheduled__2024-03-26T00:00:00+00:00 [queued]>
[2024-03-27T00:00:00.261+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_catchup_backfill_v02.first_task scheduled__2024-03-26T00:00:00+00:00 [queued]>
[2024-03-27T00:00:00.262+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 6
[2024-03-27T00:00:00.268+0000] {taskinstance.py:2217} INFO - Executing <Task(BashOperator): first_task> on 2024-03-26 00:00:00+00:00
[2024-03-27T00:00:00.271+0000] {standard_task_runner.py:60} INFO - Started process 38583 to run task
[2024-03-27T00:00:00.273+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag_with_catchup_backfill_v02', 'first_task', 'scheduled__2024-03-26T00:00:00+00:00', '--job-id', '73', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_catup_and_backup.py', '--cfg-path', '/tmp/tmp0pawstx3']
[2024-03-27T00:00:00.275+0000] {standard_task_runner.py:88} INFO - Job 73: Subtask first_task
[2024-03-27T00:00:00.300+0000] {task_command.py:423} INFO - Running <TaskInstance: dag_with_catchup_backfill_v02.first_task scheduled__2024-03-26T00:00:00+00:00 [running]> on host 0277b3465f8e
[2024-03-27T00:00:00.342+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ethan_wu' AIRFLOW_CTX_DAG_ID='dag_with_catchup_backfill_v02' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2024-03-26T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-26T00:00:00+00:00'
[2024-03-27T00:00:00.343+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-03-27T00:00:00.343+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world, this is the first task!']
[2024-03-27T00:00:00.348+0000] {subprocess.py:86} INFO - Output:
[2024-03-27T00:00:00.350+0000] {subprocess.py:93} INFO - hello world, this is the first task!
[2024-03-27T00:00:00.350+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-03-27T00:00:00.363+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=dag_with_catchup_backfill_v02, task_id=first_task, execution_date=20240326T000000, start_date=20240327T000000, end_date=20240327T000000
[2024-03-27T00:00:00.379+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-27T00:00:00.389+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
